app/src/main/res/drawable/ic_history_empty.xml

```diff
@@ -0,0 +1,206 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
that's a large fcking svg, have you seen the warning from AS?

`Very long vector path (1432 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector`

Reply:
on all big pictures with vectors we have this info message